src/ExpressionEvaluator/CSharp/Test/ResultProvider/ValueFormattingTests.cs

```diff
@@ -94,12 +94,18 @@ public void Char()
                     case '\t':
                         expected = "\\t";
                         break;
+                    case '\f':
```

Comment:
This file made BeyondCompare unhappy.  Maybe confirm that the encoding didn't change.